src/content/team/team.json

```diff
@@ -3,72 +3,100 @@
     {
       "name": "Da-Jin Chu",
       "role": "Technical Director",
-      "profileImage": "./profileImages/dajin.jpg"
+      "profileImage": "./profileImages/dajin.jpg",
+      "email": "chu.daj@husky.neu.edu",
+      "linkedIn": "https://www.linkedin.com/in/dajinchu/"
```

Comment:
You may be able to generalize this field for all social medias by making it an object with keys that are social media names and values that are the links. This way we can easily add more different ones (portfolios, instagram, etc) if we want in the future.